docs/spec/spec.md

```diff
@@ -2177,6 +2171,12 @@ used to handle errors, and they are not necessarily the last interceptor in the
 chain. However, in the case of there is no error interceptors in the pipeline, pipeline returns the internal error 
 response to the client similar to any HTTP service resource.
 
+##### 8.1.1.4 Getting JWT information
```

Comment:
Need to add this section to the table of contents

Reply:
done